src/System.Buffers.ReaderWriter/System/Buffers/Writer/BufferWriterT.cs

```diff
@@ -19,6 +19,8 @@ public BufferWriter(T output)
             _span = output.GetSpan();
         }
 
+        public ReadOnlySpan<byte> NewLine;
```

Comment:
Public field?

Reply:
yeah, I copied/pasted it from the non-generic writer. It should be a property. I will change.